### DISCUSSION TITLE
[@types/eslint-scope] version

### DISCUSSION THREAD

**Discussion by rlucan** [DefinitelyTyped/DefinitelyTyped#58121] [Issues with a @types package]
Is there a chance to change the latest version of @types/eslint-scope? Seems like our company's artifactory expects that for a given package the related @types/package has the same version. But as there is no eslint-scope@3.7.2 the artifactory does not proxy the @types/eslint-scope@3.7.2 as well. Would be nice to have @types/eslint-escope at least 4.0.0.

The npm i output:
```
npm ERR! Not Found - GET http://artifactory.company.com:80/artifactory/api/npm/virtual-npm/@types/eslint-scope/-/eslint-scope-3.7.2.tgz
npm ERR! 404
npm ERR! 404  '@types/eslint-scope@http://artifactory.company.com:80/artifactory/api/npm/virtual-npm/@types/eslint-scope/-/eslint-scope-3.7.2.tgz' is not in this registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
```

[a comment by an automated account is not shown]

**Comment by leumasme** [comment]
Seems like only types for eslint-scope 3.7 are available. The latest 3.7.* release was 3.7.3 which is now 7 years old.
The more up to date 8.3.0 release of the package has about 10x more weekly downloads than 3.7.3 so would be nice if the types got updated for newer versions.

**Comment by hkleungai** [comment]
I have opened a PR https://github.com/DefinitelyTyped/DefinitelyTyped/pull/72627 to upgrade to v8.3 typings. Feel free to take a look if you find it helpful to you.
